vulnerable_people_form/form_pages/address_lookup.py

```diff
@@ -75,15 +75,19 @@ def _create_test_address(postcode):
 
 @form.route("/address-lookup", methods=["POST"])
 def post_address_lookup():
-    session["form_answers"] = {
-        **session.setdefault("form_answers", {}),
-        "support_address": {**json.loads(request_form()["address"])},
-    }
+
     session["error_items"] = {}
 
     if not validate_address_lookup():
         return redirect("/address-lookup")
 
+    session["form_answers"] = {
+        **session.setdefault("form_answers", {}),
+        "support_address": {**json.loads(request_form()["address"])},
+    }
+    postcode = {**json.loads(request_form()["address"])}.get("postcode", None)
```

Comment:
seeing as request_form()["address"] gets parsed into a dict on lines 86 and 88 and 91, is it worth having it done once and stored in a variable?

Reply:
Agree with @VivianAllen

Reply:
Sure